backgroundShapes.js

```diff
@@ -0,0 +1,173 @@
+function backgroundShapes() {
+  // If user asked for reduced motion while browsing, don't run the script
+  if (window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
+    return;
+  }
+
+  // Get the Canvas element from the DOM
+  const canvas = document.querySelector('.background-shapes');
```

Comment:
Maybe use a separate selector for JS or a data attribute ?

Reply:
Done